.env

```diff
@@ -7,6 +7,9 @@ OTEL_EXPORTER_OTLP_ENDPOINT=http://otelcol:4317
 OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=${OTEL_EXPORTER_OTLP_ENDPOINT}
 OTEL_EXPORTER_OTLP_METRICS_ENDPOINT=${OTEL_EXPORTER_OTLP_ENDPOINT}
 
+# Metrics Temporality
+OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE=cumulative
```

Comment:
Do we want delta to be the default?

Reply:
The spec default for this var is `cumulative` https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk_exporters/otlp.md#additional-configuration

Reply:
As @puckpuck said, I'm using `cumulative` as the default since that is what the spec defines. Also since the default setup is exporting to Prometheus, I think we should keep it at cumulative. :slightly_smiling_face: 

The `prometheus` collector exporter used by this demo [does drop delta histograms.](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/d6c3f1acc69eb40161b678ff94afaab2efb50481/exporter/prometheusexporter/accumulator.go#L238-L241) so we might have missing data there if we'd set it to delta by default.